lib/Test/Tutorial.pod

```diff
@@ -509,7 +509,7 @@ turn it into a real test.
 
 =head2 Testing with taint mode.
 
-Taint mode is a funny thing.  It's the globalest of all global
+Taint mode is a funny thing.  It's the most global of all global
```

Comment:
I feel like this specific correction missed the point. This was a joke/pun, and its intent was to be humerus while conveying the idea. I feel like changing it to correct grammar misses the point and reduces the desired impact of the statement.

Reply:
ok, I guess I didn't get the joke. reverted.